tests/session/transaction-integration-002.phpt

```diff
@@ -61,9 +61,7 @@ try {
     ] );
     $manager->executeCommand(DATABASE_NAME, $cmd, ['session' => $sessionB]);
 } catch (MongoDB\Driver\Exception\CommandException $e) {
-    $rd = $e->getResultDocument();
-
-    echo (isset($rd->errorLabels) && in_array("TransientTransactionError", $rd->errorLabels)) ?
+    echo $e->hasErrorLabel('TransientTransactionError') ?
```

Comment:
I had written these, but forgotten to ``git add`` them.

Reply:
If I understand correctly, this test was only using `executeCommand()` so that you could inspect the error label by calling `CommandException::getResultDocument()`. I think this should be rewritten to execute a bulk write and then call `hasErrorLabel` on the BulkWriteException.

That may not be possible in this PR, as `phongo_execute_bulk_write()` doesn't appear to be considering the reply document when throwing its exception. If so, please link a new ticket to PHPC-1222 to populate error labels for bulk writes and make a note about updating this test.